doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/examples/ml-example.md

```diff
@@ -1,10 +1,253 @@
+We collect some helpful links for users who are getting started with a managed Kubernetes service.
+
+#### Landing pages
+Landing pages for the major cloud providers' Kubernetes services can be found at the following links.
+- [AKS landing page (Azure)](https://azure.microsoft.com/en-us/services/kubernetes-service/)
+- [EKS landing page (Amazon Web Services)](https://aws.amazon.com/eks/)
+- [GKE landing page (Google Cloud)](https://cloud.google.com/kubernetes-engine)
+
+#### General documentation
+For documentation on managed Kubernetes services, see the following links.
+- [AKS docs](https://docs.microsoft.com/en-us/azure/aks/)
+- [EKS docs](https://docs.aws.amazon.com/eks/latest/userguide/)
+- [GKE docs](https://cloud.google.com/kubernetes-engine/docs/)
+
+#### Setting up node pools
+Running the example workload in this guide requires setting up a pool or group of Kubernetes nodes.
+To learn about node group or node pool setup with managed Kubernetes services, see
+the following links.
+- [AKS node pools](https://docs.microsoft.com/en-us/azure/aks/use-multiple-node-pools)
+- [EKS node groups](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html)
+- [GKE node pools](https://cloud.google.com/kubernetes-engine/docs/concepts/node-pools)
+
+### Set up a node pool for the XGBoost benchmark
+
+For the workload in this guide, it is recommended to use a pool or group of Kubernetes nodes
+with the following properties:
+- 10 nodes total
+- A capacity of 16 CPU and 64 Gi memory per node. For the major cloud providers, suitable instance types include
+    * m5.4xlarge (Amazon Web Services)
+    * Standard_D5_v2 (Azure)
+    * e2-standard-16 (Google Cloud)
+- Each node should be configured with 1000 gigabytes of disk space (to store the training set).
+
+### Optional: Set up an autoscaling node pool
+
+**If you would like to try running the workload with autoscaling enabled**, use an autoscaling
+node group or pool with
+- 1 node minimum
+- 10 nodes maximum
+
+The 1 static node will be used to run the Ray head pod. This node may also host the KubeRay
+operator and Kubernetes system components. After the workload is submitted, 9 additional nodes will
+scale up to accommodate Ray worker pods. These nodes will scale back down after the workload is complete.
+
+## Deploying the KubeRay operator
+
+Once you have set up your Kubernetes cluster, deploy the KubeRay operator:
+```shell
+kubectl create -k "github.com/ray-project/kuberay/ray-operator/config/default?ref=v0.3.0-rc.0"
+```
+
+## Deploying a Ray cluster
+
+Now we're ready to deploy the Ray cluster that will execute our workload.
+
+:::{tip}
+The Ray cluster we'll deploy is configured such that one Ray pod will be scheduled
+per 16-CPU Kubernetes node. The pattern of one Ray pod per Kubernetes node is encouraged, but not required.
+Broadly speaking, it is more efficient to use a few large Ray pods than many small ones.
+:::
```

Comment:
TODO in a follow-up: Move to a best practices page elsewhere and link.

Reply:
Once we move it, we should also at least give a hint at why that's the case (shared memory can be leveraged more effectively, reduced inter-node communication, less Ray control plane overhead)